spec/models/user/human_spec.rb

```diff
@@ -27,6 +27,19 @@
       expect(user.errors.first.attribute).to eq(:username)
     end
 
+    context 'personal team' do
+      it 'creates personal team' do
+        user = users(:alice).dup
+        user.username = "Alice2"
+        user.save!
+        personal_team = user.personal_team
+        expect(user.personal_team).to be_present
+        expect(personal_team.teammember?(user)).to eq(true)
+        expect(personal_team.private?).to eq(true)
+      end
+
+    end
+
```

Comment:
maybe add test to make sure personal team is being removed when user is deleted?